external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -116,8 +117,9 @@ impl Interpreter {
             call_stack: CallStack::new(),
             runtime_limits_config: loader.vm_config().runtime_limits_config.clone(),
         };
-
-        profile_open_frame!(gas_meter, function.pretty_string());
+        move_vm_profiler::gas_profiler_feature_enabled! {
```

Comment:
This is not needed, however was added as requested by Ashok as an extra protection in case the feature tree somehow became broken, to ensure we don't write data to the profiler in a half state, and "break" in a predictable way. I think you have a point that these calls in particular are redundant, given that the macro will not get built if the flag in the profiler crate is off, so I will remove this.